### ISSUE TITLE
Trend line won't appear when there are 2 aggregations

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#48034]
### Describe the bug

Seems that we won't show the trend line if there are 2 aggregations on a chart, even though it should be possible

### To Reproduce

1) create a question like 
![image](https://github.com/user-attachments/assets/0ad5e087-70f2-4ac0-b216-7ddfa0a9fc40)

2) then enable the trend line
![image](https://github.com/user-attachments/assets/50861233-0770-4774-bff0-a9673cb34be7)

### Expected behavior

Trend line should be shown

### Logs

No logs whatsoever

### Information about your Metabase installation

```JSON
v50.26
```

### Severity

P1'ish?

### Additional context

If trend line is not supported we should simply disable it

**Comment by npfitz** [comment]
This can be simplified a bit (the filter isn't necessary to repro), but we likely shouldn't show the trend line option on questions that contain breakouts.

**Comment by cdeweyx** [comment]
Agreed with the above, we should hide this setting when there are multiple series
